### DISCUSSION TITLE
SPX v1.16.0-rc1 - Testing & Call Outs

### DISCUSSION THREAD

**Discussion by Reiers** [filecoin-project/lotus#8860] [Show and tell]
![banner_fvm2](https://user-images.githubusercontent.com/63351350/173351114-a1022684-327a-4d6d-9583-76b83e1f6794.png)

### Hello SPX Fellows
Update your mainnet SPs to Lotus [v1.16.0-rc1](https://github.com/filecoin-project/lotus/releases/tag/v1.16.0-rc1).
Export this environment "lotus daemon" shell:

`export LOTUS_USE_FVM_TO_SYNC_MAINNET_V15=1
`

:rocket: This post if you are participating in this testing round.
If you aren't able to join for reasons, please reach out to [@jennijuju](https://filecoinproject.slack.com/team/U016Q49G5B6) or me on Slack!

## Regular Operations
Your daily SP operations should stay :white_check_mark:

**Reply with:**
- [ ] - :repeat: If `lotus daemon` stays in sync
- [ ] - :seal:If you have successfully sealed some new sectors.
- [ ] - :handshake:If you have successfully made some storage deals.
- [ ] - :fire: If you have successfully made some snap deals.
- [ ] - :file_cabinet: If you have successfully setup and config the Index Provider.
- [ ] - :open_file_folder: If you have successfully had some retrieval deals.
- [ ] - :computer: If you passed your first WindowPoSt.

:exclamation: Create [an issue](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fbug&template=bug_report.yml) if you think there is a bug, don't forget to attach the logs!
   - Save the daemon logs, miner logs, and `lotus-miner info all` if you want to report a bug! If the log files are large, you can upload them in `SPX - v1.16.1-rc1 Logs` folder and link it in your issue report/slack message/..
## New feature and Improvements

- Built-in actor [bundles](https://github.com/filecoin-project/builtin-actors)

Read more about it [here.](https://github.com/filecoin-project/lotus/releases/tag/v1.16.0-rc1)

**Comment by TippyFlitsUK** [comment]
- [x] - :repeat: If `lotus daemon` stays in sync
- [x] - :seal:If you have successfully sealed some new sectors.
- [x] - :handshake:If you have successfully made some storage deals.
- [ ] - :fire: If you have successfully made some snap deals.
- [x] - :file_cabinet: If you have successfully setup and config the Index Provider.
- [x] - :open_file_folder: If you have successfully had some retrieval deals.
- [x] - :computer: If you passed your first WindowPoSt.

**Comment by beck-8** [comment]
don't set variables, `export LOTUS_USE_FVM_TO_SYNC_MAINNET_V15=1`,will cause forks
The problem is here,[slack](https://filecoinproject.slack.com/archives/C01D42NNLMS/p1655212806247079?thread_ts=1655191868.760769&cid=C01D42NNLMS)
- [ ]  - 🔁 If lotus daemon stays in sync
- [ ]  - 🦭If you have successfully sealed some new sectors.
- [ ]  - 🤝If you have successfully made some storage deals.
- [ ]  - 🔥 If you have successfully made some snap deals.
- [ ]  - 🗄️ If you have successfully setup and config the Index Provider.
- [ ]  - 📂 If you have successfully had some retrieval deals.
- [ ]  - 💻 If you passed your first WindowPoSt.

**Reply by Reiers** [reply]
This is the mainnet assignment - you should ` export LOTUS_USE_FVM_TO_SYNC_MAINNET_V15=1`
If calibration then dont use the env var.

This is the calib test:
https://github.com/filecoin-project/lotus/discussions/8862
